config.example.yaml

```diff
@@ -6,7 +6,7 @@ googleAnalytics: '' # todo: set your analytics id.
 
 params:
   image: /images/banner.png
-  logo: /images/logo.png
+  logo: logo.png # todo: add your custom logo to static/
```

Comment:
I think it should be `/logo.png` instead of `logo.png` :thinking:

Reply:
It's working both ways for me when deploying at the root level. However, I changed to a subdirectory deployment, and it breaks with `/logo.png`. I am guessing this is related to the topic we're discussing on #40?